### DISCUSSION TITLE
Gallery does not strictly sort after "Date taken" !

### DISCUSSION THREAD

**Discussion by sintopist** [FossifyOrg/General-Discussion#400] [General]
Hi,

I noticed that Gallery does not sort images strictly by "date taken". 

Explanation:

I had copied hundrets of images from an extarnal SD card onto my new phone. Now, Gallery seems to consider the date, when I copied the files, as being the "date taken", while the files are actually taken many years back. 

Why is that? and how can I change this?

The image properties show for example 
"Last modified 2024-09-21, 16:58"
which actually is the date/time when I just only copied the file onto the phone (but without modifiying the image at all !), and 

"Date taken 2017-12-14, 21:42"

The annoying effect is, that the images are not sorted by the truely date taken, if . So older photos appear at top of the list or the grid. 

How can i change/correct that?

Thanks !!

Best

**Comment by Aga-C** [comment]
See: https://github.com/FossifyOrg/Gallery/issues/296#issuecomment-2362851257.
